<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Öğrenci Bilgilerini Alma Arayüzü</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grp_egitim">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>50</y>
     <width>231</width>
     <height>121</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="title">
    <string>Eğitim</string>
   </property>
   <widget class="QRadioButton" name="rd_lisans">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lisans</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_lisanustu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Lisansüstü</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_sinif">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Sınıf</string>
   </property>
   <widget class="QRadioButton" name="rd_sinif1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.Sınıf</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_sinif2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2.Sınıf</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_sinif3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3.Sınıf</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_sinif4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>4.Sınıf</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_zorunluderslerLisan">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Alınana Zorunlu Dersler</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="ch_AIIT">
      <property name="text">
       <string>AİİT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ch_TurkDili">
      <property name="text">
       <string>Türk Dili</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="ch_ISG">
      <property name="text">
       <string>İSG</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="grp_Durum">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Durum</string>
   </property>
   <widget class="QRadioButton" name="rd_Ders">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ders</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_Tez">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tez</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_ZorunluDerslerLisansustu">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>410</y>
     <width>171</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Alınana Zorunlu Dersler</string>
   </property>
   <widget class="QCheckBox" name="ch_SecmeliDersler">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seçmeli Dersler</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_Seminer">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seminer</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ch_Tez">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tez</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>71</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>Ad :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>Soyad :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>Bölüm :</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>139</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="ln_ad"/>
    </item>
    <item>
     <widget class="QLineEdit" name="ln_soyad"/>
    </item>
    <item>
     <widget class="QLineEdit" name="ln_bolum"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>240</y>
     <width>171</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QPushButton" name="btn_BilgileriYazdir">
      <property name="text">
       <string>Bilgileri Yazdır</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_MezuniyetSorgula">
      <property name="text">
       <string>Mezuniyet Sorgula</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_OgrenciDurumYazdir">
      <property name="text">
       <string>Öğrenci Durum Yazdır</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
